.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "audio_common"]
+	path = audio_common
```

Comment:
Maybe put it in lib/audio_common

Reply:
done

Reply:
after changed the `README.md`, All checking passed now.